src/presentation/controllers/CreateCustomerController.ts

```diff
@@ -0,0 +1,30 @@
+import { Response, Request } from "express";
+import httpStatus from "http-status";
+import { injectable, inject } from "tsyringe";
+import { IRouterController } from "@routes/Controller";
+import { tokens } from "@/di/tokens";
+import { ICustomerService } from "@/domain/customer/types/services/ICustomerService";
+import { ICustomerInfo } from "@/domain/customer/types/Customer";
+
+@injectable()
+export class CreateCustomerController implements IRouterController {
+  constructor(
+    @inject(tokens.CustomerService)
+    private customerService: ICustomerService
+  ) {
+    this.customerService = customerService;
```

Comment:
Essa linha é necessária? Não daria para chamar o this.customerService direto na linha 22?